commodore/config.py

```diff
@@ -28,6 +28,9 @@ def __init__(self, api_url=None, api_token=None, global_git=None, verbose=False,
         self._verbose = verbose
         self.username = username
         self.usermail = usermail
+        self.local = None
+        self.push = None
+        self.interactive = None
```

Comment:
These were assigned from the cli.py file but not formally defined as part of the type Config. This is the definition of the fields (at least, one of the ways... I'm happy to change that should someone have a strong opinion on another way to make the fields part of the type definition :) )